rlclientlib/schema/v2/ActionDictionary.fbs

```diff
@@ -0,0 +1,8 @@
+namespace reinforcement_learning.messages.flatbuff.v2;
+
+table ActionDictionary {
```

Comment:
similar to comment about "action". Do we want to have some less specific structure here - i.e. DedupInfo or something like this (which may have only action_dictionary for now)? It will help to have generic_event interface look more generic + give us more space for further development.

Reply:
I see your point about naming it in a more generic way.
One thight though is that I rather keep this struct dedup specific for now.

Reply:
Renamed.